extract_token_from_browser.py

```diff
@@ -1,64 +1,176 @@
 #!/usr/bin/env python3
+# Examples:
+# python ./extract_token_from_browser.py chrome --profile="Profile 2" \
+#   --token=xoxc-foo-bar
+# python ./extract_token_from_browser.py firefox-snap --token=xoxc-baz
 
 import argparse
 import json
-from pathlib import Path
+import os
+import secretstorage
 import sqlite3
 import sys
 
+from pathlib import Path
+from Crypto.Cipher import AES
+from Crypto.Protocol.KDF import PBKDF2
```

Comment:
This means pycryptodome (or pycrypto) has to be installed. Might want to add a try/except around it so the script still works for fetching from Firefox even without this library.